.go-version

```diff
@@ -1,2 +1,2 @@
-1.91.2
```

Comment:
i merged with `1.91.2`, which turned out to be taken, so i lazily published it as `1.91.3`. thus the jump to .4